src/setuptools_scm/version.py

```diff
@@ -267,7 +267,7 @@ def simplified_semver_version(version):
             )
 
 
-def release_branch_semver(version):
+def release_branch_semver_version(version):
```

Comment:
changing the now exposed name unfortunately constitutes a breaking change, and would need a rename shim + a deprecation warning so the old name can be removed in 5.0

Reply:
Ok, no problem I'll just revert it.